ova/Libraries/provision-opendistro.sh

```diff
@@ -0,0 +1,306 @@
+        exit 1;
+    fi
+}
+
+## Show script usage
+getHelp() {
+
+   echo ""
+   echo "Usage: $0 arguments"
+   echo -e "\t-d   | --debug Shows the complete installation output"
+   echo -e "\t-i   | --ignore-health-check Ignores the health-check"
+   echo -e "\t-h   | --help Shows help"
+   exit 1 # Exit script after printing help
+}
+
+
+## Install the required packages for the installation
+installPrerequisites() {
+    logger "Installing all necessary utilities for the installation..."
+    eval "$sys_type install curl unzip wget libcap -y -q $debug"
+    echo -e '[AdoptOpenJDK] \nname=AdoptOpenJDK \nbaseurl=http://adoptopenjdk.jfrog.io/adoptopenjdk/rpm/centos/$releasever/$basearch\nenabled=1\ngpgcheck=1\ngpgkey=https://adoptopenjdk.jfrog.io/adoptopenjdk/api/gpg/key/public' | eval "tee /etc/yum.repos.d/adoptopenjdk.repo $debug"
+    eval "$sys_type install adoptopenjdk-11-hotspot -y -q $debug"
+    export JAVA_HOME=/usr/
+
+    if [  "$?" != 0  ]
+    then
+        echo "Error: Prerequisites could not be installed"
+        exit 1;
+    else
+        logger "Done"
+    fi
+}
+
+## Add the Wazuh repository
+addWazuhrepo() {
+
+    logger "Adding the Wazuh repository..."
+    eval "rpm --import https://packages.wazuh.com/key/GPG-KEY-WAZUH $debug"
+    if [ "${STATUS_PACKAGES}" = "prod" ]; then
+      logger "Adding production repository..."
+      # Temporal repository
+      eval "echo -e '[wazuh_trash]\ngpgcheck=1\ngpgkey=https://packages-dev.wazuh.com/key/GPG-KEY-WAZUH\nenabled=1\nname=EL-$releasever - Wazuh\nbaseurl=https://packages-dev.wazuh.com/trash/yum/\nprotect=1' | tee /etc/yum.repos.d/wazuh_pre.repo $debug"
+      #eval "echo -e '[wazuh_repo]\ngpgcheck=1\ngpgkey=https://packages-dev.wazuh.com/key/GPG-KEY-WAZUH\nenabled=1\nname=EL-$releasever - Wazuh\nbaseurl=https://packages.wazuh.com/3.x/yum/\nprotect=1' | tee /etc/yum.repos.d/wazuh.repo $debug"
+    elif [ "${STATUS_PACKAGES}" = "dev" ]; then
+      logger "Adding development repository..."
+      # Temporal repository
+      eval "echo -e '[wazuh_trash]\ngpgcheck=1\ngpgkey=https://packages-dev.wazuh.com/key/GPG-KEY-WAZUH\nenabled=1\nname=EL-$releasever - Wazuh\nbaseurl=https://packages-dev.wazuh.com/trash/yum/\nprotect=1' | tee /etc/yum.repos.d/wazuh_pre.repo $debug"
+      #eval "echo -e '[wazuh_pre-release]\ngpgcheck=1\ngpgkey=https://packages-dev.wazuh.com/key/GPG-KEY-WAZUH\nenabled=1\nname=EL-$releasever - Wazuh\nbaseurl=https://packages-dev.wazuh.com/pre-release/yum/\nprotect=1' | tee /etc/yum.repos.d/wazuh.repo $debug"
+    fi
+
+    if [  "$?" != 0  ]
+    then
+        echo "Error: Wazuh repository could not be added"
+        exit 1;
+    else
+        logger "Done"
+    fi
+}
+
+## Wazuh manager
+installWazuh() {
+
+
+    logger "Installing the Wazuh manager..."
+    eval "$sys_type install wazuh-manager-${WAZUH_VERSION} -y -q $debug"
+    if [  "$?" != 0  ]
+    then
+        echo "Error: Wazuh installation failed"
+        exit 1;
```

Comment:
```suggestion

    logger "Installing the Wazuh manager..."
    eval "$sys_type install wazuh-manager-${WAZUH_VERSION} -y -q $debug"
    if [ "$?" != 0 ]; then
        echo "Error: Wazuh installation failed"
        exit 1;
```

Reply:
Done in 528feec208a367ba616fb87ea4237096d0681494